addons/mail/data/ir_cron_data.xml

```diff
@@ -42,5 +42,16 @@
             <field name="state">code</field>
         </record>
 
+        <record id="ir_cron_delete_rtc_sessions" model="ir.cron">
+            <field name="name">RTC Sessions: Delete RTC sessions that haven't been active for 3 minutes</field>
+            <field name="interval_number">2</field>
```

Comment:
we can't have a useless cron running forever every 2 minutes when there's absolutely nothing to do, i.e. most of the time. This must be set to a much lower frequency (possibly with cron triggers to wake it up if there is a good reason to do so, which I don't think there is)